test/test-cls-ah.ts

```diff
@@ -21,67 +21,50 @@ var semver = require('semver');
 
 if (semver.satisfies(process.version, '<8') || !process.env.GCLOUD_TRACE_NEW_CONTEXT) {
   console.log('Skipping cls-ah tests on node version without async hooks');
-  return;
-}
-
-var cls = require('../src/cls-ah'/*.js*/);
+} else {
```

Comment:
Why is this else needed?

Reply:
`tsc` doesn't allow top-level `return` statements.